python/config.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum, auto
+
+hathceries = {'BAKER LAKE HATCHERY': ('WA_HATCHERY_DATA',), 'BEAVER CREEK HATCHERY': ('WA_HATCHERY_DATA',), 'BINGHAM CREEK HATCHERY': ('WA_HATCHERY_DATA',),
```

Comment:
hathceries = {'BAKER LAKE HATCHERY': (Hatcheries.WA_HATCHERY_DATA,), 

Might have to define class Hatcheries above this